### ISSUE TITLE
Support to allow arbitrary spaces

### ISSUE THREAD

**Issue by ranvis** [bee4/robots.txt#8]
According to the draft http://www.robotstxt.org/norobots-rfc.txt BNF,
the right side of rules can have arbitrary number of spaces (or tabs.)

```
User-agent:a
Disallow: /no
Disallow:/no

User-agent:    b
Disallow:    /no
```

**Comment by shulard** [comment]
Hello,

Thanks for the feedback !
Do you want to work on a PR for that ?

**Comment by shulard** [comment]
I think it's now fixed, you just need to wait the 2.0.2 release which will be done very soon !
If you want you can review my commit and check directly on master

**Comment by ranvis** [comment]
Thank you for implementing this!
on 0c61b28, it looks like spaceless line cannot be handled yet.

``` php
require_once(__DIR__ . '/vendor/autoload.php');
$text = 'User-agent:a
Disallow:/no';
$parser = \Bee4\RobotsTxt\Parser::parse($text);
var_dump($parser->match('a', '/no')); // bool(true)
```

**Comment by shulard** [comment]
It's now fixed in the a9d7b4321c64333e89f168d32a20e5f7657c5339.

Thanks for your feedback !
